src/zmq_van.h

```diff
@@ -110,6 +111,8 @@ class ZMQVan : public Van {
     if (my_node_.id != Node::kEmpty) {
       std::string my_id = "ps" + std::to_string(my_node_.id);
       zmq_setsockopt(sender, ZMQ_IDENTITY, my_id.data(), my_id.size());
+      int hwm = 1;
```

Comment:
Why is this change necessary?

Reply:
This basically makes the amq_send calls blocking. This was necessary because otherwise the messages in the newly added priority queue will get polled immediately and pushed onto the zmq's FIFO queue. This defeats the purpose of adding a priority queue.